calc.html

```diff
@@ -254,7 +254,7 @@
 
         <tr>
         <td class="setting-label">Mining productivity bonus:</td>
-        <td>+<input id="mprod" class="mprod" type="number" step="2" value="0" min="0" onchange="changeMprod()">%</td>
+        <td>+<input id="mprod" class="mprod" type="number" step="10" value="0" min="0" onchange="changeMprod()">%</td>
```

Comment:
This change got pulled in from #163, and should probably be kept there, separate from this change.